src/javascripts/components/logoutButton.js

```diff
@@ -1,8 +1,11 @@
 import firebase from 'firebase/app';
 import 'firebase/auth';
+import { domEventsListener } from '../helpers/events/domEvents';
 
 const signMeOut = () => {
+  document.querySelector('body').removeEventListener('click', domEventsListener);
   firebase.auth().signOut();
+  window.location.reload();
```

Comment:
Did you find this was necessary and if so, why?

Reply:
Actually we tried that when we had issues with the duplicate cards but we can take it out. The removeEventListener is working.

Reply:
It it is not needed, go ahead and remove it on this PR and once that is done, you can merge!